repo2docker/buildpacks/conda/__init__.py

```diff
@@ -26,6 +26,9 @@ class CondaBuildPack(BuildPack):
 
     path = ['${CONDA_DIR}/bin']
 
+    entrypoint = ['/srv/conda/bin/jupyter']
+    command = ['notebook', '--ip', '0.0.0.0']
```

Comment:
Splitting one command across entrypoint and command seems unusual. What's the benefit?